src/components/common/HotDocumentSection.tsx

```
@@ -1,53 +1,114 @@
-          ? await documentPostApi.getWeeklyPopularDocumentPost()
-          : await documentPostApi.getDailyPopularDocumentPost();
-      setDocs(res.documentPostsPage?.content ?? []);
-    })();
-  }, [period]);
+    const fetchDocuments = async () => {
+      setLoading(true);
+      try {
+        if (activeTab === "주간") {
+          const response = await documentPostApi.getWeeklyPopularDocumentPost();
+          if (response && response.documentPostsPage && response.documentPostsPage.content) {
+            setDocuments(response.documentPostsPage.content);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)